runtime/convertValues_test.go

```diff
@@ -104,62 +154,72 @@ func TestExportValue(t *testing.T) {
-				},
-				[]interpreter.Value{}...),
+			valueFactory: func(inter *interpreter.Interpreter) interpreter.Value {
+				return interpreter.NewArrayValue(
+					inter,
+					interpreter.VariableSizedStaticType{
+						Type: interpreter.PrimitiveStaticTypeAnyStruct,
+					},
+					common.Address{},
+				)
+			},
 			expected: cadence.NewArray([]cadence.Value{}),
 		},
 		{
 			label: "Array (non-empty)",
-			value: interpreter.NewArrayValueUnownedNonCopying(
-				interpreter.VariableSizedStaticType{
-					Type: interpreter.PrimitiveStaticTypeAnyStruct,
-				},
-				[]interpreter.Value{
+			valueFactory: func(inter *interpreter.Interpreter) interpreter.Value {
+				return interpreter.NewArrayValue(
+					inter,
+					interpreter.VariableSizedStaticType{
+						Type: interpreter.PrimitiveStaticTypeAnyStruct,
+					},
+					common.Address{},
 					interpreter.NewIntValueFromInt64(42),
 					interpreter.NewStringValue("foo"),
-				}...,
-			),
+				)
+			},
 			expected: cadence.NewArray([]cadence.Value{
 				cadence.NewInt(42),
 				cadence.String("foo"),
 			}),
 		},
 		{
 			label: "Dictionary",
-			value: interpreter.NewDictionaryValueUnownedNonCopying(
-				newTestInterpreter(t),
-				interpreter.DictionaryStaticType{
-					KeyType:   interpreter.PrimitiveStaticTypeString,
-					ValueType: interpreter.PrimitiveStaticTypeAnyStruct,
-				},
-			),
+			valueFactory: func(inter *interpreter.Interpreter) interpreter.Value {
+				return interpreter.NewDictionaryValue(
+					inter,
+					interpreter.DictionaryStaticType{
+						KeyType:   interpreter.PrimitiveStaticTypeString,
+						ValueType: interpreter.PrimitiveStaticTypeAnyStruct,
+					},
+				)
+			},
 			expected: cadence.NewDictionary([]cadence.KeyValuePair{}),
 		},
 		{
 			label: "Dictionary (non-empty)",
-			value: interpreter.NewDictionaryValueUnownedNonCopying(
-				newTestInterpreter(t),
-				interpreter.DictionaryStaticType{
-					KeyType:   interpreter.PrimitiveStaticTypeString,
-					ValueType: interpreter.PrimitiveStaticTypeAnyStruct,
-				},
-				interpreter.NewStringValue("a"),
-				interpreter.NewIntValueFromInt64(1),
-				interpreter.NewStringValue("b"),
-				interpreter.NewIntValueFromInt64(2),
-			),
+			valueFactory: func(inter *interpreter.Interpreter) interpreter.Value {
+				return interpreter.NewDictionaryValue(
+					inter,
+					interpreter.DictionaryStaticType{
+						KeyType:   interpreter.PrimitiveStaticTypeString,
+						ValueType: interpreter.PrimitiveStaticTypeAnyStruct,
+					},
+					interpreter.NewStringValue("a"),
+					interpreter.NewIntValueFromInt64(1),
+					interpreter.NewStringValue("b"),
+					interpreter.NewIntValueFromInt64(2),
+				)
+			},
 			expected: cadence.NewDictionary([]cadence.KeyValuePair{
-				{
-					Key:   cadence.String("a"),
-					Value: cadence.NewInt(1),
-				},
 				{
 					Key:   cadence.String("b"),
 					Value: cadence.NewInt(2),
 				},
+				{
+					Key:   cadence.String("a"),
+					Value: cadence.NewInt(1),
+				},
```

Comment:
For some tests, the order of the results changed, as dictionaries are now backed by atre ordered map's, which do not provide insertion order guarantee (instead it is based on the hash input and hash algorithm)